lua/lspconfig/server_configurations/snyk_ls.lua

```diff
@@ -0,0 +1,24 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'snyk-ls' },
+    root_dir = util.find_git_ancestor,
+    single_file_support = true,
+    filetypes = {},
```

Comment:
filetypes shouldn't empty

Reply:
Snyk supports many languages, including infra-as-code languages (e.g. Terraform, Helm/Kubernetes manifests, AWS Cloudformation).

I looked at language-agnostic LSPs for inspiration, like [diagnostic-languageserver](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/server_configurations/diagnosticls.lua).

Is there a better way to handle this case, like `filetype = {"*"}`?

Or do you recommend specifying every filetype?

How do you handle something like AWS Cloudformation which could be JSON or YAML?

Reply:
I think it's better remove `filetype` and add config example in `doc.describe`.

Reply:
Done

Reply:
Done